cmd/containerd/command/main.go

```diff
@@ -193,6 +193,13 @@ can be used and modified as necessary as a custom configuration.`
 			"revision": version.Revision,
 		}).Info("starting containerd")
 
+		if config.SchedCore {
+			log.G(ctx).Info("enabling sched core support")
+			if err := os.Setenv("SCHED_CORE", "1"); err != nil {
```

Comment:
Could this be configured by the task manager and explicitly set into the shim environment?